### DISCUSSION TITLE
Auto Animate isn't being imported

### DISCUSSION THREAD

**Discussion by rootedsoftware** [remix-run/remix#5187] [Q&A]
### What version of Remix are you using?

1.7.0

### Steps to Reproduce

I created a brand new app using create react app and another one using the vite create tool. Both of these worked just fine with the @formkit/auto-animate package installed. 
 I then created a new remix app using the create tool and then installed the auto-animate package ```npm install @formkit/auto-animate``` After installing I tried to run ```npm run dev``` and I get this error

```
The path "@formkit/auto-animate" is imported in app/routes/index.tsx but "@formkit/auto-animate" was not found in your node_modules. Did you forget to install it?
```

I tried following the instructions here https://github.com/formkit/auto-animate/issues/9 by adding    ```serverDependenciesToBundle: ['@formkit/auto-animate/react']``` to the remix.config.js file with no luck. I also posted my testing in that issue and let them know I'd cross post here.

I do have it installed in the package.json

```
"@formkit/auto-animate": "^1.0.0-beta.3",
```

Reproduction here https://github.com/Rooted-Software/remix-auto-animate-reproduction

### Expected Behavior

Load and build without an issue as it does on other React projects

### Actual Behavior

During the building I get this message

```Building Remix app in production mode...
The path "@formkit/auto-animate" is imported in app/routes/index.tsx but "@formkit/auto-animate" was not found in your node_modules. Did you forget to install it?```

**Comment by martinmckenna** [comment]
did you figure out how to get the console message to go away? I had this issue with another package and I added it to `serverDependenciesToBundle` and my app works now, but the message persists
